Userland/Applications/ImageViewer/main.cpp

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
- *
+ * Copyright (c) 2021, Mohsan Ali <mohsan0073@gmail.com>
+ * 
```

Comment:
Extra space at the end of this line
```suggestion
 *
```